staging/src/k8s.io/apiserver/pkg/apis/audit/v1/types.go

```diff
@@ -72,17 +72,27 @@ const (
 type Event struct {
 	metav1.TypeMeta `json:",inline"`
 
-	// AuditLevel at which event was generated
+	// Audit level at which event was generated. Valid values include: 'None',
+	// 'Metadata', 'Request' and 'RequestResponse'.
 	Level Level `json:"level" protobuf:"bytes,1,opt,name=level,casttype=Level"`
 
 	// Unique audit ID, generated for each request.
 	AuditID types.UID `json:"auditID" protobuf:"bytes,2,opt,name=auditID,casttype=k8s.io/apimachinery/pkg/types.UID"`
 	// Stage of the request handling when this event instance was generated.
+	// The value can be one of:
+	// - 'RequestReceived': The stage for events generated as soon as the audit handler receives
+	//   the request, and before it is delegated down the handler chain.
+	// - 'ResponseStarted': The stage for events generated once the response headers are sent,
+	//   but before the response body is sent. This stage is only generated for long-running requests
+	//   (e.g. watch).
+	// - 'ResponseComplete': The stage for events generated once the response body has been completed,
+	//   and no more bytes will be sent.
+	// - 'Panic': The stage for events generated when a panic occurred.
 	Stage Stage `json:"stage" protobuf:"bytes,3,opt,name=stage,casttype=Stage"`
 
-	// RequestURI is the request URI as sent by the client to a server.
+	// The request URI as sent by the client to a server.
```

Comment:
isn't `requestURI is the request ...` the expected style, i.e. the JSON field name? Has the suggested style changed?

Reply:
The "expected style" is a little bit vague. We can say `requestURI is the request ...` if all developers are comfortable with it. If we are more aggressive at it, we should say ```The `requestURL` is the request...```. The second style is much clearer when the field name is a regular English word, e.g. ``` `request` is the request received ...```. However, I'm not sure our developers are comfortable with adding extra backticks. The "style" adopted by this PR is actually a compromise between developer habits and API consumers.

Reply:
To be honest, most API documentation comments use UpperCamelCase names. Making a change to this feels like it'd be a bigger change than this PR.

I'd like to find a way to have the end-users see relevant API docs (and for the OpenAPI to use the right capitalization, too) - but that does feel like a KEP-sized work package.